src/main/groovy/jenkins/automation/rest/RestApiJobManagement.groovy

```diff
@@ -0,0 +1,123 @@
+// Credit: https://github.com/sheehan/job-dsl-gradle-example
+
+package jenkins.automation.rest
+
+import groovyx.net.http.HttpResponseDecorator
+import groovyx.net.http.RESTClient
+import javaposse.jobdsl.dsl.*
+import org.apache.http.entity.ContentType
+
+class RestApiJobManagement extends MockJobManagement {
+
+    final RESTClient restClient
+
+    RestApiJobManagement(String baseUrl) {
+        if (!baseUrl.endsWith("/")) {
+            baseUrl += "/"
+        }
+        restClient = new RESTClient(baseUrl)
+        restClient.handler.failure = { it }
```

Comment:
> For closures accepting a single argument, the keyword it can be used to represent that single parameter rather than explicitly specifying the single parameter's name

http://www.javaworld.com/article/2074349/core-java/groovy-s-special-words.html